build.sh

```diff
@@ -1,8 +1,75 @@
-git clone https://github.com/ufcg-lsd/asperathos-controller ./controller/asperathos-controller
-docker build controller/ -t asperathos_controller
-git clone https://github.com/ufcg-lsd/asperathos-monitor ./monitor/asperathos-monitor
-docker build monitor/ -t asperathos_monitor
-git clone https://github.com/ufcg-lsd/asperathos-manager ./manager/asperathos-manager
-docker build manager/ -t asperathos_manager
-git clone https://github.com/ufcg-lsd/asperathos-visualizer ./visualizer/asperathos-visualizer
-docker build visualizer/ -t asperathos_visualizer
+#!/bin/bash
+
+helpFunction()
+{
+   echo ""
+   echo "Usage: $0 -b BRANCH [-r] [-l] [-h]"
+   echo -e "\t-b BRANCH      Select Branch        The branch you want to clone from (the same branch will be used for all projects). If you use -b with a branch different from the current one the files will be deleted and the new branch will be downloaded"
+   echo -e "\t-r             Reset                If you want to reset the current asperathos files"
```

Comment:
Which files does this refer to? The config files? All code/files? Specify in order to make it clear.